tests/smoke-tests/pom.xml

```diff
@@ -1318,6 +1318,31 @@
                      </args>
                   </configuration>
                </execution>
+               <execution>
+                  <phase>test-compile</phase>
+                  <id>upgrade-current-version-phrase1</id>
+                  <goals>
+                     <goal>create</goal>
+                  </goals>
+                  <configuration>
+                     <noWeb>false</noWeb>
+                     <instance>${basedir}/target/upgrade/currentVersion</instance>
+                     <args>
+                        <arg>--disable-persistence</arg>
+                     </args>
+                  </configuration>
+               </execution>
+               <execution>
+                  <phase>test-compile</phase>
+                  <id>upgrade-current-version-phase2</id>
+                  <goals>
+                     <goal>upgrade</goal>
+                  </goals>
+                  <configuration>
+                     <instance>${basedir}/target/upgrade/currentVersion</instance>
+                  </configuration>
+               </execution>
+
```

Comment:
This is being inserted in the middle of existing executions used together for a test, separating them...moving this addition up (e.g line 1306) or down (e.g. line 1379) would make it more understandable later.